main.css

```diff
@@ -0,0 +1,97 @@
+.heading {
+    font-family: 'Apple-Chancery', cursive;
+    font-style: italic;
+    font-size: 60px;
+}
+
+ body {
+    background-image: url("background.png");
+    background-size: cover;
+}
+
+.whiteFont {
+    color: white;
+}
+.yellow {
+    color: yellow;
+}
+
+.red {
+    color: #C91A44;
+}
+
+.Italic {
+    font-style: italic;
+}
+
+.bordercontainer {
+    /*border: 3px solid blue;*/
+    /*display: flex;*/
+    margin-right: 29%;
```

Comment:
it is not typical to use percentages for margins and paddings, it works but it is not very standard. You can probably achieve the same using em.